shared-services/providers.tf

```diff
@@ -12,3 +12,13 @@ provider "aws" {
   # profile = "cool-sharedservices-account-admin"
   region = var.aws_region
 }
+
+# Read-only AWS Organizations provider
+provider "aws" {
+  alias = "organizationsreadonly"
+  default_tags {
+    tags = var.tags
+  }
```

Comment:
Just for my curiosity: If this is a read-only provider does setting tags do anything?

Reply:
It doesn't do anything, but I think it's a good idea to have the habit of always applying tags to the providers so that they always flow down to any resources that are created.